src/__tests__/Login.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
+
+import { Provider } from 'react-redux';
+import store from '../app/store';
+import App from '../App';
+import GlobalContextProvider from '../app/contexts';
+import * as firebaseMock from '../app/firebase-configs';

```

Comment:
this is not a mock, it's the real module, even if you're planning to intercept it with `spyOn` later.

Reply:
I'm solving this conversation with the caveat that it should be improved in the future.